### PR TITLE
Merge dependency updates to Staging

### PR DESCRIPTION
**Pull request by kevinlind**
* Update dependencies to 3.0.0 versions.

* Use H4 headers for code examples in getting-started

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/edgemedia/build.gradle.kts [hunk 1]
```diff
@@ -31,21 +31,10 @@ aepLibrary {
...
 
-    androidTestImplementation("com.adobe.marketing.mobile:edge:$mavenEdgeVersion-SNAPSHOT"){
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-        exclude(group = "com.adobe.marketing.mobile", module = "edgeidentity")
-    }
-    // Update it to 3.+ after release
-    androidTestImplementation("com.adobe.marketing.mobile:edgeidentity:3.0.0-SNAPSHOT"){
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-    }
+    androidTestImplementation("com.adobe.marketing.mobile:edgeidentity:3.0.0")
```
Comment (emdobrin) [inline code/edgemedia/build.gradle.kts:37]: for androidTestImplementation should this be edge and edgeidentity?

Comment (kevinlind) [reply code/edgemedia/build.gradle.kts:37]: Edge is a dependency for the project, so it's automatically a dependency to the tests as well.

Comment (emdobrin) [reply code/edgemedia/build.gradle.kts:37]: are there any errors if we explicitly specify that?

Comment (cacheung) [reply code/edgemedia/build.gradle.kts:37]: I don't see any error when include
 androidTestImplementation("com.adobe.marketing.mobile:edge:$mavenEdgeVersion")

**Review by cacheung — APPROVED** [review]

**Pull request merged**